# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libpcap/limits.h.patch.musl
# Copyright (C) 2019 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- libpcap-1.9.0/pcap-usb-linux.c.vanilla	2019-09-18 11:46:58.100000000 +0200
+++ libpcap-1.9.0/pcap-usb-linux.c	2019-09-18 11:47:22.988000000 +0200
@@ -57,6 +57,7 @@
 #include <sys/ioctl.h>
 #include <sys/mman.h>
 #include <sys/utsname.h>
+#include <limits.h>
 #ifdef HAVE_LINUX_USBDEVICE_FS_H
 /*
  * We might need <linux/compiler.h> to define __user for
